spec/cnb/monthly_rates_spec.rb

```diff
@@ -27,11 +27,11 @@
 
     context 'valid parameters' do
       it 'returns rate for given currency and date' do
-        expect(monthly_rates.rate_for(currency, Date.today)).to eql(19.564)
+        expect(monthly_rates.rate_for(currency, Date.today)).to eql(0.19564)
       end
 
       it 'returns rate for given currency with default date' do
-        expect(monthly_rates.rate_for(currency)).to eql(19.564)
+        expect(monthly_rates.rate_for(currency)).to eql(0.19564)
```

Comment:
so this is tested on JPY? Because thats only currency i know which has 100 quantity..